### ISSUE TITLE
The machine isn't improving

### ISSUE THREAD

**Issue by KaspervanM** [KaspervanM/RaceGame#7]
The neural network mutates and natural selection takes place, but no big improvements are made yet.
This should be looked at.

**Comment by KaspervanM** [comment]
I think making natural selection more strict can improve it slightly, but variation in models will decrease.

EDIT
It will cause a phenomenon called the genetic bottleneck effect. This can mess with the genetic variation.

**Comment by evanpacini** [comment]
I think you should do the opposite, decrease natural selection to increase model variety. The winning model isn't necessarily the one you suppose 😜

**Comment by KaspervanM** [comment]
Yes, indeed my judgement may be clouded, but results aren't. If those who are unfit remain, the next generation will have less new mutated models, since those who are selected to live will stay alive the next round. They won't be replaced.

EDIT
Now exactly half of them die and half of them live.

EDIT
Some might survive a hundred generations, while others were less lucky with their mutations and die after only one.
